### DISCUSSION TITLE
Whitespace tokens to match in extras

### DISCUSSION THREAD

**Discussion by ahelwer** [tree-sitter/tree-sitter#1137] [Q&A] [answered]
Looking through various grammars, almost all of them match whitespace with some variant of the `/\s|\r?\n/` pattern. Quite a few of them also match some unicode space characters, for example `\uFEFF` or `\u00A0`. Is there a good list of such unicode space characters to match? Or is it very much the sort of thing that should be handled case by case per language?

**Comment by tek** [comment, marked as answer]
I'm using `/\p{Zs}/`
